src/types/Synergism.d.ts

```diff
@@ -590,12 +590,16 @@ export interface Player {
     octeractUpgrades: Record<keyof typeof octeractData, OcteractUpgrade>
     dailyCodeUsed: boolean
     hepteractAutoCraftPercentage: number
+
+    octeractHideToggle: boolean
+    octeractBuyMaxToggle: boolean | 'ANY'
```

Comment:
can this type be changed to 'ENABLED' | 'DISABLED' | 'ANY' rather than having 2 different types?

Reply:
Done :)